lib/grandpa/vote_message.go

```diff
@@ -153,7 +153,7 @@ func (s *Service) validateMessage(from peer.ID, m *VoteMessage) (*Vote, error) {
 		return nil, err
 	}
 
-	// check that setIDs match
+	// vote is considered invalid if set ID do not match.
```

Comment:
nit the comment is not really needed I think

```suggestion
```

Reply:
addressed